src/modules/MinifyImage/Module.js

```diff
@@ -82,8 +82,8 @@ module.exports = function MinifyImage(options, UI) {
     }
 
 
-    function output(image, datauri, mimetype, wasmSuccess) {
-      step.output = { src: datauri, format: mimetype, wasmSuccess, useWasm: options.useWasm };
+    function output(datauri, mimetype) {
```

Comment:
Hmm, removing wasmSuccess -- can we get a review from @Divy123 ? Thank you!

Reply:
`wasmSuccess` was from my PR. Why was it removed? Does it not use PixelManipulation? Can you just pass `false` instead of passing nothing?

Reply:
No, it doesnt use pixel manipulation

Reply:
Okay. Could you pass `false` instead of nothing?

Reply:
@HarshKhandeparkar  Done!